src/CSET/operators/plot.py

```diff
@@ -467,3 +504,115 @@ def plot_line_series(
     _make_plot_html_page(plot_index)
 
     return cube
+
+
+def plot_vertical_line_series(
+    cube: iris.cube.Cube,
+    filename: str = None,
+    series_coordinate: str = "pressure",
+    sequence_coordinate: str = "time",
+    # line_coordinate: str = "realization",
+    **kwargs,
+) -> iris.cube.Cube:
+    """Plot a line plot against a type of vertical coordinate.
+
+    A 1D line plot with y-axis as pressure coordinate can be plotted, but if the sequence_coordinate is present
+    then a sequence of plots will be produced.
+
+
+    The cube must be 1D.
+
+    Parameters
+    ----------
+    cube: Cube
+        Iris cube of the data to plot. It should have a single dimension.
+    filename: str, optional
+        Name of the plot to write, used as a prefix for plot sequences. Defaults
+        to the recipe name.
+    series_coordinate: str, optional
+        Coordinate about which to make a series. Defaults to ``"pressure"``. This
+        coordinate must exist in the cube.
+    sequence_coordinate: str, optional
+        Coordinate about which to make a plot sequence. Defaults to ``"time"``.
+        This coordinate must exist in the cube.
+
+    Returns
+    -------
+    Cube
+        The original cube (so further operations can be applied).
+
+    Raises
+    ------
+    ValueError
+        If the cube doesn't have the right dimensions.
+    TypeError
+        If the cube isn't a single cube.
+    """
+    # Ensure we've got a single cube.
+    cube = _check_single_cube(cube)
+
+    try:
+        coord = cube.coord(series_coordinate)
+    except iris.exceptions.CoordinateNotFoundError as err:
+        raise ValueError(f"Cube must have a {series_coordinate} coordinate.") from err
+
+    if cube.ndim > 1:
```

Comment:
Should this `if` condition be here? It currently doesn't check that you have the needed coordinate if you only have 1 coord. See below for an example of what I think you are trying to do:
https://github.com/MetOffice/CSET/blob/775bc9587983321a81a2821e6786b5def07918f5/src/CSET/operators/plot.py#L481-L487

Reply:
The the cube is only 1D then it should only have a series coordinate such as pressure for the vertical levels and not a sequence coordinate such as time for the slider. So I think I need to keep this check.

Reply:
@jfrost-mo I have added more infromation into the plot function for what I think it should test. Essentially I want it to give an error message if the series coordinate does not exist in cube with 1 or more dimensions. If the cube has more than 1 dimension I want to ensure that the additional dimension is a sequence coordinate,